packages/models/src/lib/implementation/schemas.ts

```diff
@@ -7,7 +7,12 @@ import {
 } from './limits';
 import { filenameRegex, slugRegex } from './utils';
 
-export const primitiveValueSchema = z.union([z.string(), z.number()]);
+export const primitiveValueSchema = z.union([
+  z.string(),
+  z.number(),
+  z.boolean(),
+  z.null(),
```

Comment:
FYI, all the cell contents are uploaded to portal as strings (makes it simpler), [the `tableToGQL` function](https://github.com/code-pushup/cli/blob/main/packages/core/src/lib/implementation/report-to-gql.ts#L111-L135) converts the content using `.toString()`.

So for booleans this would print `true` or `false`, while nulls would require extra handling. But it might be better for each plugin to take care of it's own string conversion, because it has additional context to know how best to display the information than what we can do generically. E.g. maybe boolean values would be better formatted as `yes`/`no` or `passed`/`failed` or `✅`/`❌`, while null values could be better formatted as empty string or `-` or `n/a`.

Reply:
Thanks for the info!